bears/haskell/HaskellLintBear.py

```diff
@@ -45,7 +54,7 @@ def process_output(self, output, filename, file):
             for other_lines in range(1, len(from_lines)):
                 assert from_lines[other_lines] == to_lines[other_lines]
             line_nr = issue['startLine']
-            line_to_change = file[line_nr-1]
+            line_to_change = file[line_nr - 1]
```

Comment:
unnecessary change

Reply:
@manankalra Since, coala PEP8 , this is a suitable change.

Reply:
but it isn't related to the issue you're working on
_different issue, different commit_ - not here ;)